tests/common.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+"""
+<Program Name>
+  common.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Feb 6, 2018
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Common code for in-toto unittests, import like so:
+  `import tests.common`
```

Comment:
It might help if you explained how this module should be used.

Namely, that test cases inherit from CliTestCase:
```
class MyTestCase(tests.common.CliTestCase):
```

Do you also want to mention the provided convenience/mocking methods?

Reply:
I think that the class and method docstrings of `CLITestCase` describes its usage well enough.